src/theme/MuiThemeProvider/muiTheme.ts

```diff
@@ -192,5 +193,21 @@ export default createTheme({
         },
       },
     },
+    MuiTooltip: {
```

Comment:
Rather then override styles at the base level, we can override them with styles in the component directory